### ISSUE TITLE
Remove dependency on Wrench npm package (it is deprecated)

### ISSUE THREAD

**Issue by pulekies** [jonlb/node-build#4]
**What did you expect to happen?**
Installing 'node-build' via npm should succeed with no warnings. 

**What actually happens**
Installing 'node-build' via npm gives the following warning:
`npm WARN deprecated wrench@1.3.9: wrench.js is deprecated! You should check out fs-extra (https://github.com/jprichardson/node-fs-extra) for any operations you were using wrench for.`

**How to reproduce**
Run an npm install for 'node-build':
`npm install build`

**Comment by graingert** [comment]
@pulekies I'm presuming you found this via Sinon: https://github.com/sinonjs/sinon/commit/37ff38b7369c569501bb94e684eac6f5db7bd556

build was included by accident.

**Comment by pulekies** [comment]
Yes, thanks for responding. It was suspicious to me that it was included, but I figured I might as well log this issue anyway.

**Comment by graingert** [comment]
@jonlb it might be worth issuing:

```sh
npm run deprecate build "did you mean 'npm run build'?"
```

**Comment by trollepierre** [comment]
wrench is deprecated...
npm WARN deprecated wrench@1.3.9: wrench.js is deprecated! You should check out fs-extra (https://github.com/jprichardson/node-fs-extra) for any operations you were using wrench for. Thanks for all the usage over the years.
